runtime/codert_vm/CMakeLists.txt

```diff
@@ -72,3 +72,8 @@ target_link_libraries(j9codert_vm
 		j9vm_interface
 		j9vm_compiler_defines
 )
+
+target_enable_ddr(j9codert_vm j9ddr)
```

Comment:
Neither `j9codert_vm` nor `compiler` are considered by the current DDR process: why add them?

Reply:
Because in the cmake world we have to explicitly state the headers to be scanned (a limitation im trying overcome at a later date).  On the omr side, the ddr support ignores the target output files unless the target is an executable or a shared library. Thus when you call `target_enable_ddr()` on any other kind of target, it serves only to parse the source/header files for macros

Reply:
That doesn't address my question: those modules weren't processed by DDR before, why do we want to start?